### DISCUSSION TITLE
White dot in the left corner of Dualsense's Touchpad

### DISCUSSION THREAD

**Discussion by Ditrey** [ndeadly/MissionControl#482] [Q&A] [answered]
Greetings. After connecting dualsense controller to switch, the touch pad is surrounded by blue light and a dot of light in the left down corner. I am wondering if it was a malfunction or a preset for the controller. If it is the latter, is there any way by which I could change the way the lighting behaves?

![dualsensewhitedot](https://user-images.githubusercontent.com/106951537/172113103-88c63f5d-67a3-48ff-9f37-499b2aa0948d.jpg)

**Comment by ndeadly** [comment, marked as answer]
It's intentional. The white leds are used to show the same player number indicator patterns used by official switch controllers. There's currently no way to change this behaviour from within mission control, but you could build your own version with the code that sets the LEDs removed.

**Reply by Ditrey** [reply]
Ok I am going to try that at some time, thanks!
